common/laws/00_land_reform.txt

```diff
@@ -111,7 +117,11 @@ law_commercialized_agriculture = {
 	icon = "gfx/interface/icons/law_icons/commercial_agriculture.dds"
 	progressiveness = 0
 	modifier = {
-		tax_per_capita_add = 0.05
+		tax_per_capita_add = 0.1
+		interest_group_ig_industrialists_pol_str_mult = 0.15
```

Comment:
There is already a production method change, I think this is a little bit too much. I also like changing the shares of the production method more, giving more shares to the capitalists.